app/livechat/client/views/app/tabbar/visitorInfo.html

```diff
@@ -10,42 +10,45 @@
 			{{#with user}}
 				<div class="info">
 					{{#if name}}
-						<h3 title="{{name}}"><i class="status-{{status}}"></i> {{name}}</h3>
+						<h3 title="{{name}}">{{name}} <i class="status-{{status}}"></i></h3>
 						<p class="secondary-font-color">{{username}}</p>
 					{{else}}
-						<h3 title="{{username}}"><i class="status-{{status}}"></i> {{username}}</h3>
+						<h3 title="{{username}}">{{username}} <i class="status-{{status}}"></i></h3>
 					{{/if}}
 					<ul>
 						{{#if utc}}<li><i class="icon-clock"></i>{{userTime}} (UTC {{utc}})</li>{{/if}}
-						{{#each visitorEmails}} <li><i class="icon-mail"></i> {{address}}{{#if verified}}&nbsp;<i class="icon-ok success-color"></i>{{/if}}</li> {{/each}}
-						{{#each phone}} <li><i class="icon-phone"></i> {{phoneNumber}}</li> {{/each}}
-						{{#if lastLogin}} <li><i class="icon-calendar"></i> {{_ "Created_at"}}: {{createdAt}}</li> {{/if}}
-						{{#if lastLogin}} <li><i class="icon-calendar"></i> {{_ "Last_login"}}: {{lastLogin}}</li> {{/if}}
+						{{#each visitorEmails}}<li><i class="icon-mail"></i> {{address}}{{#if verified}}&nbsp;<i class="icon-ok success-color"></i>{{/if}}</li> {{/each}}
+						{{#each phone}}<li><i class="icon-phone"></i> {{phoneNumber}}</li> {{/each}}
+						{{#if lastLogin}}<li><i class="icon-calendar"></i> {{_ "Created_at"}}: {{createdAt}}</li> {{/if}}
+						{{#if lastLogin}}<li><i class="icon-calendar"></i> {{_ "Last_login"}}: {{lastLogin}}</li> {{/if}}
 						{{#if ip}}<li><i class="icon-laptop"></i><span>{{ip}}</span></li>{{/if}}
 						{{#if os}}<li><i class="{{osIcon}}"></i><span>{{os}}</span></li>{{/if}}
 						{{#if browser}}<li><i class="{{browserIcon}}"></i><span>{{browser}}</span></li>{{/if}}
+						{{#if customVisitorFields}}
+							{{#each customVisitorFields}}<li><strong>{{label}}</strong>: {{value}}</li>{{/each}}
+						{{/if}}
 					</ul>
 				</div>
 			{{/with}}
 
-			{{#with room}}
-				<div class="info">
-					<ul>
-						{{#if facebook}} <li><i class="icon-facebook"></i>{{_ "Facebook_Page"}}: {{facebook.page.name}}</li> {{/if}}
-						{{#if sms}} <li><i class="icon-mobile"></i>{{_ "SMS_Enabled"}}</li> {{/if}}
-						{{#if topic}} <li><strong>{{_ "Topic"}}</strong>: {{{RocketChatMarkdown topic}}}</li> {{/if}}
-						{{#if tags}} <li><strong>{{_ "Tags"}}</strong>: {{joinTags}}</li> {{/if}}
-					</ul>
-				</div>
-			{{/with}}
-
-			{{#with department}}
-				<div class="info">
-					<ul>
-						{{#if name}} <li>{{_ "Department"}}: {{name}}</li> {{/if}}
-					</ul>
-				</div>
-			{{/with}}
+			<div class="info">
+				<h3>{{_ "Conversation"}}</h3>
```

Comment:
I think this is good, but using `<h2>` instead of `<h3>` to downgrade is not very good, because the semantics of the HTML code are violated. I changed tag for first title and style for them. Ok?

Result
![image](https://user-images.githubusercontent.com/19654261/71233783-64781500-2318-11ea-8c8b-f4aefcd9d407.png)